package.json

```diff
@@ -44,36 +44,31 @@
     "babel-cli": "6.18.0",
     "babel-core": "6.20.0",
     "babel-eslint": "7.1.1",
+    "babel-jest": "^18.0.0",
```

Comment:
There appears to be an open issue about this for Yarn, https://github.com/yarnpkg/yarn/issues/2193